quickwit-cli/src/cli.yaml

```diff
@@ -196,8 +196,42 @@ subcommands:
         settings:
             - ArgRequiredElseHelp
         display_order: 2
-        about: Operations (add, delete, describe...) on splits.
+        about: Operations (list, add, delete, describe...) on splits.
         subcommands:
+            - list:
+                about: List all splits of an index.
+                args:
+                    - metastore-uri:
+                        about: Location of the metastore
+                        long: metastore-uri
+                        value_name: METASTORE URI
+                        required: true
+                        env: QUICKWIT_METASTORE_URI
+                    - index-id:
+                        about: Index ID
+                        long: index-id
+                        value_name: INDEX ID
+                        required: true
+                    - states:
+                        about: Comma-separated list of SplitState to filter on.
+                        long: states
+                        value_name: SPLIT STATES
+                        multiple_occurrences: true
+                        use_delimiter: true
+                    - from:
```

Comment:
Why don't we use the same names as the search: `start_timestamp` and `end_timestamp`?

Reply:
Or since the command also supports dates, `start_date` and `end_date`?

Reply:
`from` & `to` seemed shorter an user-friendlier to me. I am more with `start-date` & `end-date` from your suggestion